tests/suites/0_stateless/05_ddl/05_0007_ddl_drop_user_v2.sql

```diff
@@ -0,0 +1,14 @@
+set enable_planner_v2=1;
+DROP USER 'test-j'@'localhost'; -- {ErrorCode 2201}
+DROP USER IF EXISTS 'test-j'@'localhost';
+
+
+CREATE USER 'test-j'@'localhost' IDENTIFIED BY 'password';
+DROP USER 'test-j'@'localhost';
+DROP USER IF EXISTS 'test-j'@'localhost';
+
+CREATE USER 'test-l'@'localhost' IDENTIFIED WITH sha256_password BY 'password';
+DROP USER 'test-l'@'localhost';
+DROP USER IF EXISTS 'test-l'@'localhost';
+DROP USER IF EXISTS 'test-l'@'localhost';
+DROP USER IF EXISTS 'test-l'@'localhost';
```

Comment:
It would be better to teardown with `set enable_planner_v2=0`;

Reply:
We has already migrate create user to planner v2. And in 05_0007_ddl_drop_user.sql the test is enable_planner_v2=0. I think if we migrate all statement to planner v2 we can delete old test.